src/main/java/org/eolang/lints/LtByXsl.java

```
@@ -98,15 +101,26 @@ public Collection<Defect> defects(final XML xmir) {
                     String.format("No severity reported by %s", this.rule)
                 );
             }
+            final Optional<String> message = xml.element("message").text();
             defects.add(
                 new DfContext(
                     new Defect.Default(
                         this.rule,
                         Severity.parsed(sever.get()),
                         new ObjectName(xmir).get(),
                         this.lineno(xml),
-                        xml.text().get(),
-                        LtByXsl.experimental(xml)
+                        message.orElseGet(() -> xml.text().get()),
+                        LtByXsl.experimental(xml),
+                        message.map(
+                                ignored ->
+                                    new Fix(
+                                        new File(
+                                            xmir.xpath("/@source/text()").get(0),
+                                            new Edit(xml.element("edit"))
+                                        )
+                                    )
+                            )
+                            .orElseGet(Fix::new)
```

@Marat-Tim maybe we should throw an exception in this case? `Fix::new` only will make things more complicated